src/transaction/creation_common.rs

```diff
@@ -1611,14 +1612,15 @@ pub(super) mod invocable_entity {
         let runtime = transaction_runtime::get(matches);
 
         let entry_point = session_entry_point::get(matches).unwrap_or_default();
-        let transferred_value = transferred_value::get(matches)
-            .map(|value| value.parse::<u64>().unwrap())
-            .unwrap_or_default();
+        let maybe_transferred_value =
+            transferred_value::get(matches).map(|value| value.parse::<u64>().unwrap());
```

Comment:
We should remove this unwrap and parse it similarly to other args in case someone provides an invalid string.

Maybe add a `parse` method that would also handle the validation below.

Reply:
Agreed. How about this magic one-liner:
```suggestion
            transferred_value::get(matches).and_then(|value| value.parse::<u64>().ok());
```

Reply:
But probably better to add an error variant to report invalid integer rather than panic (or ignore it as in the one-liner above)